src/io/test_files.rs

```diff
@@ -15,6 +15,19 @@ pub enum ListTestsFilesError {
 	WalkDir(String, #[source] walkdir::Error),
 }
 
+/// List all cairo test files
+///
+/// The given `&Path` will be walked through to find all cairo test files
+/// according to `TEST_FILE_REGEX` (`r"^test_.*\.cairo$"`)
+///
+/// Returns a vector of all test files found
+///
+/// # Examples
+/// ```ignore
+/// for path in list_test_files(std::path::Path::new("."))? {
+///     assert_eq!(path.to_str().unwrap().ends_with(".cairo"), true)
+/// }
+/// ```
```

Comment:
This exemple can not be run because it has no file system to read from, but it can at least be written in a way that allow it to compile.

https://doc.rust-lang.org/rustdoc/write-documentation/documentation-tests.html#attributes
use the `no_run` attribute rather than the `ignore` one there